### PR TITLE
Run on FloydHub integration

### PR DESCRIPTION
**Pull request by ReDeiPirati**
Hi @lukaszkaiser , here's the Run on FloydHub integration.

- Added the button (and the badge) in the `README.md`
- Added the `floyd.yml` config file to specify the environment
- Added the `floyd_requirements.txt` file to automatically install tensor2tensor in the environment

I've changed the data and checkpoint paths of the examples in the `README.md` from `~/` (HOME mapping) to `./` (current working directory) to improve usability. Sounds good for you?

Looking forward to your feedback :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,15 +35,21 @@ installs T2T, downloads MNIST, trains a model and evaluates it:
 ```
 pip install tensor2tensor && t2t-trainer \
   --generate_data \
-  --data_dir=~/t2t_data \
-  --output_dir=~/t2t_train/mnist \
+  --data_dir=./t2t_data \
```
Comment (lukaszkaiser) [inline README.md]: Let's not change the home directory here, it's really better to make this in ~.

File: README.md [hunk 2]
```diff
@@ -34,15 +35,21 @@ installs T2T, downloads MNIST, trains a model and evaluates it:
...
+  --output_dir=./t2t_train/mnist \
   --problem=image_mnist \
   --model=shake_shake \
   --hparams_set=shake_shake_quick \
   --train_steps=1000 \
   --eval_steps=100
 ```
 
+### Run on FloydHub
```
Comment (lukaszkaiser) [inline README.md]: Could you move this section to the end, before papers? Thanks!

File: floyd.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+env: tensorflow-1.9
```
[the remark this answers is not in the document]
Comment (ReDeiPirati) [reply floyd.yml]: Done!

File: README.md [hunk 4]
```diff
@@ -9,6 +9,7 @@ welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg)](CO
 [![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/tensor2tensor/Lobby)
 [![License](https://img.shields.io/badge/License-Apache%202.0-brightgreen.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Travis](https://img.shields.io/travis/tensorflow/tensor2tensor.svg)](https://travis-ci.org/tensorflow/tensor2tensor)
+[![Run on FH](https://static.floydhub.com/button/button-small.svg)](https://floydhub.com/run?template=https://github.com/tensorflow/tensor2tensor)
```
[the remark this answers is not in the document]
Comment (ReDeiPirati) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (ReDeiPirati) [reply README.md]: Done!

**Comment by ReDeiPirati** [comment]
@lukaszkaiser @rsepassi thanks for the feedback! I have just updated the PR.

**Pull request merged**